appveyor.yml

```diff
@@ -49,7 +49,7 @@ install:
 test_script:
     - cd c:\projects\workspace
     - vendor\bin\phpunit --prepend=tests/xdebug-filter.php --coverage-clover=clover.xml --coverage-xml=coverage/coverage-xml --log-junit=coverage/junit.xml
-    - php bin\infection --threads=4 --log-verbosity=none --coverage=coverage
+    - php bin\infection --threads=%NUMBER_OF_PROCESSORS% --log-verbosity=none --coverage=coverage --no-progress --only-covered --ignore-msi-with-no-mutations
```

Comment:
slight change, let's see if that works

Reply:
why do we need `--only-covered` here, and in travis as well?

Reply:
To make it faster, and on Travis it was already there (and is only for the PRs)